app/lib/actions/katello/organization/manifest_refresh.rb

```diff
@@ -61,6 +61,7 @@ def humanized_name
 
         def finalize
           org = ::Organization.find(input[:organization_id])
+          subject_organization.clear_syspurpose_status(cached: false) if subject_organization.simple_content_access?
```

Comment:
the 'cached' param is on the simple_content_access method :)

Reply:
Moved, prob a copy pasta fail :)